server/portal/apps/workspace/api/views.py

```diff
@@ -82,30 +85,6 @@ def _get_app(app_id, app_version, user):
     return data
 
 
-def _test_listing_with_existing_keypair(system, user):
-    # TODOv3: Add Tapis system test utility method with proper error handling https://jira.tacc.utexas.edu/browse/WP-101
```

Comment:
I think we should probably move this comment to the `test_system_credentials` function